README.md

```diff
@@ -37,3 +37,22 @@ You can also run the tests and watch in a browser with:
 ```bash
 npx playwright test --ui
 ```
+
+## Running these tests on CI
+
+The E2E test suite is currently run on the [Accredited Programmes UI
+repository](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui)
+[as part of
+CI](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/blob/4b70ab1045f12f127e03738ada18f0d68458443d/.circleci/config.yml#L204).
+
+The user is currently set to use an email address of one of the team members.
+This is something we should address as soon as possible, and replace with a
+shared service account. The username and password are set as GitHub Actions
+secrets in this repository. When run as part of CI, they are set in the CircleCI
+(our current CI tool) project settings.
```

Comment:
GitHub Actions is also CI, no? Would it be clearer to say that this repository uses GitHub Actions for its CI checks, and takes its credentials from GitHub Actions secrets. When run as part of CI in the UI repository, the credentials are read from the CircleCI project settings?